risc0/zkp/src/prove/mod.rs

```diff
@@ -64,10 +64,14 @@ where
     C: CircuitDef<F>,
     CS: CircuitStepHandler<F::Elem>,
 {
+    // The taps are the entries of the trace used in evaluating the constraints.
     let taps = circuit.get_taps();
 
+    // The number of columns used for encoding zkvm control instructions.
     let code_size = taps.group_size(RegisterGroup::Code);
+    // The number of columns used for encoding the execution trace.
     let data_size = taps.group_size(RegisterGroup::Data);
+    // The number of columns used for the PLOOKUP argument.
```

Comment:
Is this for PLOOKUP? My understanding is this is for PLONK permutation arguments.